### ISSUE TITLE
Quantile loss for predictions

### ISSUE THREAD

**Issue by jz312** [jdb78/pytorch-forecasting#392]
Hi, 
For the forecasting problem SKU future volume using TFT given in the tutorial, two types of predictions are generated: 
- point forecast: predictions = best_tft.predict(val_dataloader)
- quantile forecast: raw_predictions, x = best_tft.predict(val_dataloader, mode="raw", return_x=True), and raw_predictions['prediction'] are quantiles

It is unclear to me how one quantile loss function can generate both types of predictions. My understanding is that there is no change in TFT architecture or loss function (QuantileLoss) in generating both predictions. 

I would expect to change the loss function to MSE (or similar) as well as training for point forecasts. Did I miss something?

**Comment by jdb78** [comment]
Quantile forecasts are still point forecasts. The 50% percentile / median is obtained by training for a MAE. The Quantile forecast simple forecasts multiple quantiles including the 50% percentile. Of course, you can directly use MAE or other metrics such as MSE, SMAPE, MASE, etc.
